MetadataExtractor/Util/ByteTrie.cs

```diff
@@ -90,7 +90,7 @@ public void Add(T value, params byte[][] parts)
         }
 
         /// <summary>Store the given value at the specified path.</summary>
-        internal void Add(T value, ReadOnlySpan<byte> part)
+        public void Add(T value, ReadOnlySpan<byte> part)
```

Comment:
Is the byte[] overload still used?

Does this need to be added to the public API files?

Reply:
This was a brand new member to avoid allocation, the params byte[] array overload still used.

Reply:
Is there a diagnostic here about the public member not being in the API files?

Reply:
Annoying, this analyzer randomly stops reporting for me in Visual Studio, and I have to catch these manually. Updated!

Reply:
Weird. There should at least be a build warning, and CI should fail. I'll take a look some time.